web/src/OverviewActionBar.tsx

```diff
@@ -574,6 +585,11 @@ export function FilterTermField({ termFromUrl }: { termFromUrl: FilterTerm }) {
       <SrOnly component="label" htmlFor={FILTER_FIELD_ID}>
         Filter resource logs by text or /regexp/
       </SrOnly>
+      <FilterTermInfoTooltip
+        id={FILTER_FIELD_TOOLTIP_ID}
+        title="RegExp should be wrapped in forward slashes, is case-insensitive, and parsed in JavaScript."
```

Comment:
@ellenkorbes i'm not exactly sure how to describe the flavor of regexp that we're using. 😅 do you have any suggestions?

Reply:
if you can put a link in the tooltip, linking to a js regexp reference would both be useful and make the specific wording here less important

Reply:
done!

Reply:
this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

Reply:
yup! that's what i linked to.